.github/workflows/dapr_cli.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       GOVER: 1.15
-      GOLANGCILINT_VER: 1.26.0
+      GOLANGCILINT_VER: 1.31.0
```

Comment:
will you add linter action step as well?

Reply:
will do ....